spark-bukkit/src/main/java/me/lucko/spark/bukkit/folia/FoliaSupport.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of spark.
+ *
+ *  Copyright (c) lucko (Luck) <luck@lucko.me>
+ *  Copyright (c) contributors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package me.lucko.spark.bukkit.folia;
+
+/**
+ * Compatibility util for Paper's threaded regions "Folia" project.
+ */
+public enum FoliaSupport {
+    ;
+
+    public static final boolean IS_ACTIVE;
+
+    static {
+        boolean active = false;
+        try {
+            Class.forName("io.papermc.paper.threadedregions.scheduler.RegionScheduler");
```

Comment:
This class is present in paper now, too. It should rather check for `io.papermc.paper.threadedregions.RegionizedServer`.